packages/create-amplify/src/get_project_root.ts

```diff
@@ -29,10 +30,13 @@ export const getProjectRoot = async () => {
     logger.debug(`Creating directory ${projectRoot}`);
     try {
       await fsp.mkdir(projectRoot, { recursive: true });
-    } catch (e) {
-      await fsp.mkdir(path.join(process.cwd(), projectRoot), {
-        recursive: true,
-      });
+    } catch (err) {
+      if (path.isAbsolute(projectRoot)) {
+        logger.warn(
+          `Failed to create directory at ${projectRoot}. Ensure this is the correct path and you have write permissions to this location.`
+        );
+      }
+      throw err;
```

Comment:
Modifyed by @sobolk feedback.

> https://github.com/aws-amplify/amplify-backend/pull/857#discussion_r1443351730

one is a question. 

I try to write test case code.

↓ Should this be written as a test?
```
### CASE: 
throws error, when creates the project directory the user provided absolute path does not exist and define outside of current dir.

ex.) userInput = "/test"
```

```typescript
void it('creates the project root directory if the user provided absolute path does not exist and define outside of current dir.', async () => {
    process.env.npm_config_yes = 'false';
    const userInput = '/test';
    mock.method(AmplifyPrompter, 'input', () => Promise.resolve(userInput));

    const errorMsg = await getProjectRoot(); //want to receive errorMsg.
    
    //TODO: check errorMsg
  });
```

To do this, don't I need to stop the process in `logger.warn`?
ex) change `new Error(...)`